src/Our.Umbraco.Ditto/App_Data/NuGetBackup/20160415-151441/CopyLogs/UmbracoBinBackup.log

```diff
@@ -0,0 +1,82 @@
+

```

Comment:
This file can be deleted. It's from the NuGet package update. We can add an ignore rule for future reference.